src/components/TagWrapper/PTagWrapper.vue

```diff
@@ -4,7 +4,7 @@
       <template v-for="tag in tags" :key="tag.value">
         <div class="p-tag-wrapper__tag" :class="classes.tag">
           <slot name="tag" :tag="tag">
-            <PTag :value="tag" />
+            <PTag :value="tag" :small="small" />
```

Comment:
Also noticed this wasn't being passed even though it is in most upstream components